Fix: Add to Image Provenance & Signing Documents (#7164)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/provenance-verification.md

   --certificate-identity-regexp='^https://github.com/slsa-framework/slsa-github-generator/.github/workflows/generator_container_slsa3.yml@refs/tags/v[0-9]+.[0-9]+.[0-9]+$'
```

> The command will exit with `0` if the `cosign` verification was complete:

Reviewer: ```suggestion
The command will exit with `0` when the `cosign` verification is completed:
```
